tests/util/skvbc_history_tracker.py

```diff
@@ -196,6 +201,37 @@ def __repr__(self):
            f'  kvpairs={self.kvpairs}\n'
            f'  req_index={self.req_index}\n')
 
+class Status:
+    """
+    Status about the order of writes and reads.
+    This is useful for debugging and checking linearizability.
```

Comment:
This class isn't involved in checking linearizability. It's soley for debugging after a test fails.

Reply:
Indeed, let's update the Status class documentation, as suggested by Andrew.